src/utilis/colourPallatte.js

```diff
@@ -0,0 +1,10 @@
+const colourPalette = [
+	"#ffeed9",
+	"#ffc9c9",
+	"#d3ffa3",
+	"#7fd4c8",
+	"#a1a9ff",
+	"#f7c2ff",
```

Comment:
Here, just adding a blank option / white option might do the trick to add a clear color button ?This could be a shortcut :)

Reply:
sure